src/objects.rs

```diff
@@ -90,73 +90,45 @@ pub enum Nid{
     AuthorityKeyIdentifier,
 }
 
-// impl From<u32> for Nid {
-//     fn from(u: u32) -> Nid { Nid(u) }
-// }
+const OBJ_ALGO : &[u8]    = &oid!(raw 1.3.14.3.2);
```

Comment:
Agreed on all this changes, but I believe this should become something like:

```rust
pub const OID_ALGORITHM : &[u8]    = &oid!(raw 1.3.14.3.2);
```

See below for why.